todoman uses sphinx_rtd_theme to generate html docs. We remove it since
we only generate the man pages. The theme also requires a version of
node-sass, from its transitive as well as direct dependencies, which is
incompatible with the nodejs packaged in aports.

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -42,7 +42,6 @@
     "sphinx_autorun",
     "sphinx.ext.todo",
     "sphinx.ext.viewcode",
-    "sphinx_rtd_theme",
 ]
 
 source_suffix = ".rst"
@@ -63,10 +62,6 @@
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
-
-# -- Options for HTML output ----------------------------------------------
-
-html_theme = "sphinx_rtd_theme"
 
 # -- Options for manual page output ---------------------------------------
 
